lua/autorun/client/cl_restrict_sounds.lua

```diff
@@ -1,28 +1,14 @@
+local soundIsRestricted = {}
```

Comment:
It could be as simple as a `\n` delimted `.txt`, or maybe as robustaboi as a `.json`. 
What if we make a config file for cfcLib?
Has multiple sections, one of them could be "Restricted Sounds"... (_cough_ XML _cough_)

Reply:
how do i delete someone else's github